code/modules/modular_computers/file_system/programs/security/camera.dm

```diff
@@ -145,7 +145,7 @@
 		A.client.eye = A.eyeobj
 		return TRUE
 
-	if(!is_contact_area(get_area(C)))
+	if(!is_contact_area(get_area(C)) && !LAZYISIN(C.network, NETWORK_NEWS))
 		to_chat(user, SPAN_NOTICE("This camera is too far away to connect to!"))
 		return FALSE
```

Comment:
does this respect overmap distance?
so you can't transmit from offship to horizon when they're on opposite ends of the overmap

Reply:
I don't believe so and I'm not sure how to go about implementing that